packages/blockchain-extension/extension/commands/environmentConnectCommand.ts

```diff
@@ -56,6 +56,24 @@ export async function fabricEnvironmentConnect(fabricEnvironmentRegistryEntry: F
             fabricEnvironment = new FabricEnvironment(fabricEnvironmentRegistryEntry.name);
         }
 
+        let nodes: FabricNode[] = await fabricEnvironment.getNodes();
+
+        if (fabricEnvironmentRegistryEntry.url) {
+            let informOfChanges: boolean = true;
+            if (nodes.length === 0) {
+                const importNodes: boolean = await UserInputUtil.showConfirmationWarningMessage(`Error connecting to environment ${fabricEnvironmentRegistryEntry.name}: no visible nodes. Would you like to filter nodes?`);
+                if (!importNodes) {
+                    return;
+                }
+                informOfChanges = false;
```

Comment:
why do you set this back to false?

Reply:
At this stage we already asked if the user wants to edit filters, and the answer was yes. We don't want to ask again (which we would if differences were detected and informOfChanges was true) because we already decided to show the tickbox.
The tickbox will display any differences as part of the node's description (<new>/<was OLD NAME>) so the user will not miss out on information..

Reply:
ahhh i see i missed where it was being used again